### DISCUSSION TITLE
Error connection to node

### DISCUSSION THREAD

**Discussion by oscarostby** [pterodactyl/panel#5095] [Support]
Access to XMLHttpRequest at 'https://panel.mydomain.com/api/system' from origin 'https://serverip' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by Yerli-Oyuncu** [comment]
> Access to XMLHttpRequest at 'https://panel.mydomain.com/api/system' from origin 'https://serverip' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

Im getting same issue how did you fixed ?
